### PR TITLE
Added tutorial for setting up Caddy with basic examples

### PR DESCRIPTION
**Pull request by MaxJa4**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Maximilian Jannack maxijannack@t-online.de

### CODE REVIEW

File: tutorials/caddy-as-simple-reverse-proxy-and-file-server/01.en.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+---
+
+## Step 0 - `Stop any other web servers`
+
+Since Caddy will naturally use port 80 and 443 to serve HTTP and HTTPS content - as a web server does - it is best to stop any other running web servers (e.g. Nginx, Apache, ...).
+
+Please note, that depending on the OS, it is possible that a web server is already preinstalled and running without you knowing it.
+To check, you can either use
+```bash
+service --status-all
```
Comment (rook59) [inline tutorials/caddy-as-simple-reverse-proxy-and-file-server/01.en.md:57]: This does not work on all mentioned systems, some require you to use e.g. `systemctl status`. I think checking open ports is good enough.

**Review by rook59 — COMMENTED** [review]
Caddy is great, thanks for writing this tutorial!

File: tutorials/caddy-as-simple-reverse-proxy-and-file-server/01.en.md (hunk 1 above)
Comment (MaxJa4) [reply tutorials/caddy-as-simple-reverse-proxy-and-file-server/01.en.md:57]: Oh right, I was too focued on Ubuntu there. To not cause unnecessary confusion I will remove it.

**Comment by MaxJa4** [comment]
Removed the service status check not cause any confusion - should be handled safely by the port check anyway.

**Review by rook59 — APPROVED** [review]

**Pull request merged**